Mathlib/Algebra/Group/Opposite.lean

```diff
@@ -208,10 +208,9 @@ end DivInvMonoid
 theorem op_natCast [NatCast α] (n : ℕ) : op (n : α) = n :=
   rfl
 
--- See note [no_index around OfNat.ofNat]
 @[to_additive (attr := simp)]
 theorem op_ofNat [NatCast α] (n : ℕ) [n.AtLeastTwo] :
-    op (no_index (OfNat.ofNat n : α)) = OfNat.ofNat n :=
+    op (ofNat(n) : α) = ofNat(n) :=
```

Comment:
So, this is one example of fixing a theorem statement, right (as a no_index is also put on the RHS), right?

Reply:
Arguably this doesn't change the statement, only the _indexing_ of the statement. The corrections I was referring to in the description are the ones where a missing `OfNat.ofNat` is added.